Localization/de-DE/home.yaml

```
@@ -97,6 +97,6 @@ Für technische Hilfe: ",
 
 #Associated text
 #"Folding@home is now based at the Washington University in St. Louis School of Medicine, under the directorship of  Dr. Greg Bowman. Drs. John Chodera (MSKCC) and Vince Voelz (Temple University) are also active in helping manage the project. Together, their three labs are the primary drivers of Folding@home."
-"Folding@home ist derzeit bei der Washington University in St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Zusammen sind ihre drei Labore die hauptsächlichen Treiber von Folding@home."
+"Folding@home ist derzeit bei der Washington University in St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Ihre drei Labore sind die Hauptantriebskraft von Folding@home."
```

```suggestion
"Folding@home ist derzeit bei der Washington University in der St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Ihre drei Labore sind die Hauptantriebskraft von Folding@home."
```
only using "in" makes it sound like there's  a place coming
btw i really like your solution to the treiber-problem!